.travis/deploy_sbt.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o verbose
+
+# If we're on the master branch and in an application project, we should
+# build a new Docker image and push it to ECR.
+
+if [[ "$BRANCH" != "master" ]]
+then
+  echo "Not on master (BRANCH=$BRANCH); skipping deploy..."
+  exit 0
+fi
+
+if [[ "$PROJECT" == "common" ]]
+then
+  echo "Common lib doesn't have a Docker container; skipping deploy..."
+  exit 0
+fi
```

Comment:
I would pull this logic upstream into the run script – this script just deploys a given project, and leave it to the caller to determine whether it’s appropriate. Keep this script self-contained.